tls/debug.h

```diff
@@ -35,4 +35,21 @@
 
 #include "lib/log.h"
 
+#define TTLS_LOG_WITH_PEER(tls, action_expr)			\
+do {								\
+	struct sockaddr_storage _tls_peer_ss;			\
+	struct sockaddr_storage *_tls_peer_sa = &_tls_peer_ss;	\
+	if (ttls_extract_peer_addr(tls, _tls_peer_sa))		\
+		_tls_peer_sa = NULL;				\
+	action_expr;						\
+} while (0)
+
+#define TTLS_PEER_FMT "[c=%pISp] "
+#define TTLS_ERR(tls, fmt, ...) TTLS_LOG_WITH_PEER(tls, \
+	T_ERR(TTLS_PEER_FMT fmt, _tls_peer_sa, ##__VA_ARGS__))
+#define TTLS_WARN(tls, fmt, ...) TTLS_LOG_WITH_PEER(tls, \
+	T_WARN(TTLS_PEER_FMT fmt, _tls_peer_sa, ##__VA_ARGS__))
```

Comment:
@krizhanovsky TTLS messages looks like:
```
[  301.549539] [tempesta tls] Warning: [c=127.0.0.1:65280] ClientHello: bad extension size 1032 (tls->hs->ext_rem_sz=1177)
```
Does it look well or you have a better suggestion? I tried to move address part to the end but it breaks explicit `\n` convention.

Reply:
May I have a [generic] suggestion? Unless it's a debug message, it doesn't make sense to have something like internal variable or structure names in a message. It looks cryptic, and it's meaningless to a user. Still, that extra information might still be valuable to Tempesta crew when figuring out what the issue is, so, perhaps, it would make sense to replace some or all of those variable or structure members with somethig expressed with words.

In the above example, I think, **[c=127.0.0.1:65280]** can easily be replaced with just **[127.0.0.1:65280]**, and **(tls->hs->ext_rem_sz=1177)** with something like **(*remaining size* 1177)**. (I don't know what *ext_rem_sz* stands for).

On another note, perhaps, the use of round and/or square brackets in these messages could be unified.